api/proto/info/info.proto

```diff
@@ -284,9 +284,12 @@ message DevicePort {
   // DhcpConfig
   uint32 dhcpType = 11;
   string subnet = 12;
+  // gateway - OBSOLETE - obsoleted by defaultRouters
   string gateway = 13;
+  // domainname - OBSOLETE - obsoleted by dns
```

Comment:
Isn't dns about about DHCP derived status, and this domainname and dnsServers is about static config? I honestly don't know the answer.

Reply:
I see this information is being filled from NetworkPortConfig.. Not from Status..